cpp/CMakeLists.txt

```diff
@@ -70,6 +76,42 @@ if (WITH_BENCHMARK)
   add_subdirectory(benchmark)
 endif()
 
+if (WITH_JNI)
+  find_package(JNI REQUIRED)
+
+  if(NOT JNI_INCLUDE_DIRS)
+    set(JNI_INCLUDE_DIRS ${JAVA_INCLUDE_PATH})
+  endif()
+
+  add_library(milvus-storage-jni SHARED
+    src/ffi/reader_writer_jni.cpp
+    src/ffi/reader_c.cpp
```

Comment:
target `milvus-storage` already include `src/ffi/*_c.cpp`, it will cause the refined error when other target link the `milvus-storage-jni`.